src/components/author/author.twig

```diff
@@ -12,7 +12,7 @@
     {% elseif date %}
       <p>
         {# This creates a more accessible, and less confusing, user experience #}
-        <span class="u-hidden-visually">
+        <span class="u-hidden-visually" role="text">
```

Comment:
We shouldn't need the addition of `role="text"` here since it's one `span` that contains the full text. 😉 
```suggestion
        <span class="u-hidden-visually">
```

Reply:
@gerardo-rodriguez Can you elaborate on what you mean by "contains the full text?"

Reply:
@tylersticka Sure, sorry for not being clear. 😬 
```twig
{# 
  All of the text that is expected to be read together is within a single span element, so we 
  don't need to add the role="text" workaround to have it read as one whole unit of text 
#}
<span class="u-hidden-visually">
  {{ date_prefix|default('Published on') }} {{ date|date('F jS, Y') }}
</span>
```

Reply:
Am I being clear? I can try again if I still didn't explain myself well. 😅

Reply:
Okay, I think I get it. If the element is an "only child," there's no _gap_ between elements the user has to manually traverse. If there were an additional `span` after this one, then there would be.

Thank you for the explanation.